### ISSUE TITLE
Profile contains events with truncated names

### ISSUE THREAD

**Issue by Flamefire** [tensorflow/tensorflow#33814]
**System information**
- Have I written custom code: yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): RHEL 7.4
- TensorFlow installed from (source or binary): Docker container
- TensorFlow version (use command below): v2.0.0-rc2-26-g64c3d38 2.0.0
- Python version: 3.6.8
- CUDA/cuDNN version: 10.0
- GPU model and memory: K80

**Describe the current behavior**

When gathering a trace (via the supposedly only supported in TF2 method) the resulting trace file shows incomplete GPU op names. Validated in TensorBoard and deserializing the Protobuf message.
Examples:
```
GpuDevice>, long)
GpuDevice>, int)
Dimension<3>, unsigned int*)
reduced_divisor)
```

**Describe the expected behavior**

Trace shows complete function names (at least) and signatures (optional) or even better: Layers to which those activities/events belong.

**Code to reproduce the issue**

```
import datetime
import os
import tensorflow as tf
import tensorflow_datasets as tfds


def preprocess_for_mnist(image, label):
    image = tf.cast(image, tf.float32) / 255.0
    return image, label


datasets, info = tfds.load(name='mnist',
                           with_info=True,
                           as_supervised=True,
                           shuffle_files=False)

strategy = tf.distribute.OneDeviceStrategy(device='/gpu:0')

train_dataset = datasets['train'].map(preprocess_for_mnist).batch(32)

log_dir = os.path.join(os.path.expanduser('~/tf_logs'),
                       datetime.datetime.now().strftime("%Y%m%d-%H%M%S"))

with strategy.scope():
    model = tf.keras.Sequential([
        tf.keras.layers.Conv2D(32,
                               3,
                               activation='relu',
                               input_shape=(28, 28, 1)),
        tf.keras.layers.MaxPooling2D(),
        tf.keras.layers.Flatten(),
        tf.keras.layers.Dense(10, activation='softmax')
    ])

    callbacks = [
        tf.keras.callbacks.TensorBoard(log_dir=log_dir,
                                       histogram_freq=0,
                                       profile_batch=2)
    ]

    model.compile(loss='sparse_categorical_crossentropy',
                  optimizer=tf.keras.optimizers.Adam(),
                  metrics=['accuracy'])
    model.fit(train_dataset, epochs=2, steps_per_epoch=10, callbacks=callbacks)
```

**Other info / logs**

Traces generated: [train.zip](https://github.com/tensorflow/tensorflow/files/3782526/train.zip)

**Comment by qiuminxu** [comment]
I think the issue has already been fixed. The trace was collected on Oct 29th. Can you give us a trace at head, and we can investigate if the issue persists.

**Comment by Flamefire** [comment]
I tried but can't test it currently.

I just tried with the docker containers but neither tensorflow-devel-gpu-py3 nor tensorflow-devel-py3 seem to actually contain tensorflow. In `/usr/local/lib/python3.6/dist-packages/` I see some Keras_* stuff but nothing with tensorflow. `pip freeze` doesn't list it either.

Using pip to install the 2.1.0rc has upgraded the CUDA requirement to 10.1 under which the driver requires admin rights or "unrestricted mode" to enable tracing. I passed it on to our HPC admins but it might take a bit.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!

**Comment by Flamefire** [comment]
Still checking if this is an issue, @tensorflowbutler please reopen
